data/core/commands/root.lua

```diff
@@ -176,3 +176,21 @@ command.add(function()
     end
   }
 )
+
+-- double clicking the tab bar, or on the emptyview should open a new doc
+command.add(function(x, y)
+  if not x or not y then return false end
+  local node = core.root_view.root_node:get_child_overlapping_point(x, y)
+  if not node or not node:should_show_tabs() then return false end
+  local _, ty, _, th, scroll_padding = node:get_scroll_button_rect(1)
+  return ((not node.hovered_tab and node.hovered_scroll_button == 0) and y >= ty and y < ty + th)
+end, {
+  ["root:new-doc"] = function()
```

Comment:
I wonder if we should just call it `tabbar:new-doc`.

Reply:
Hrm. Would be more clear. Sure. Why not.